adlfs/tests/test_spec.py

```diff
@@ -392,7 +392,9 @@ def test_info_missing(storage, path):
 
 def test_time_info(storage):
     fs = AzureBlobFileSystem(
-        account_name=storage.account_name, connection_string=CONN_STR
+        account_name=storage.account_name,
+        connection_string=CONN_STR,
+        max_concurrency=1,
```

Comment:
Concurrent upload cannot be used for this test, otherwise `storage.insert_time` will be the timestamp of the first completed chunk and `creation_time` will be the timestamp of the finished operation (after all chunks are uploaded).